tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -41,7 +41,7 @@ public DotNetCoreDynamicInstallTest(ITestOutputHelper output) : base(output)
         [InlineData(NetCoreApp21WebApp, "2.1")]
         [InlineData(NetCoreApp31MvcApp, "3.1")]
         [InlineData(NetCoreApp50MvcApp, "5.0")]
-        [InlineData(NetCore7PreviewMvcApp, "7.0.0-preview.6.22324.4")]
+        [InlineData(NetCore7PreviewMvcApp, "7.0.0-preview.7.22375.6")]
```

Comment:
I know it wasn't like it before, but replacing `7.0.0-preview.7.22375.6` with `DotNetCoreSdkVersions.DotNet70SdkVersion` will keep us from needing to update this in the future to a new hardcoded value

Reply:
Other than that, LGTM :)